Monika After Story/game/script-topics.rpy

```diff
@@ -4846,6 +4846,32 @@ label monika_coffee:
     m 2e "It's funny how a drink for giving you energy became a treat for you to enjoy."
     m 1k "Maybe if I find out how, I'll tinker with the script so I can finally drink some! Ahaha~"
     return
+    
+    init 5 python:
+    # List of keywords for the topic.
+    for key in ['sayori','golden ending']:
```

Comment:
This topic should **only** be available if the player has gotten the good ending. The value `persistent.clearall` is True when the player has achieved that ending. Please adjust your code so the topic is only added to `monika_topics` and `monika_random_topics` if `persistent.clearall` is True.